meilisearch-http/src/index_controller/dump_actor/loaders/v1.rs

```diff
@@ -158,15 +158,12 @@ impl From<Settings> for index_controller::Settings<Unchecked> {
                 Some(None) => Setting::Reset,
                 None => Setting::NotSet
             },
-            // we previously had a `Vec<String>` but now we have a `HashMap<String, String>`
-            // representing the name of the faceted field + the type of the field. Since the type
-            // was not known in the V1 of the dump we are just going to assume everything is a
-            // String
             filterable_attributes: match settings.attributes_for_faceting {
                 Some(Some(attrs)) => Setting::Set(attrs.into_iter().collect()),
                 Some(None) => Setting::Reset,
                 None => Setting::NotSet
             },
+            sortable_attributes: Setting::NotSet,
```

Comment:
Tell me if I am wrong but I must set `NotSet` for settings that were not present in previous versions of meilisearch?